ci-scripts/rhdh-setup/create_resource.sh

```diff
@@ -353,8 +373,16 @@ get_token() {
   #shellcheck disable=SC2064
   trap "rm -rf $token_lockfile; exit" INT TERM EXIT HUP
 
+  echo "BEFORE"; cat "$token_field"
+
```

Comment:
@jhutar I assume you have this here for debugging purposes... please remove it, unless you have a specific reason to keep it here.

Reply:
Thank you Pavel. Please review now.